### ISSUE TITLE
In Metadata Editor ALT+l says you must select a track

### ISSUE THREAD

**Issue by campg2j003** [campg2j003/JAWS-Script-for-Audacity#10]
Script version: 2.1.0
Audacity version: 2.1.3
OS/Platform:Windows 10 64-bit
JAWS version: all

Description:
In the Metadata Editor pressing ALT+l (Load Template) when no tracks are in the project says "to use this feature you must select at least one track first".

Steps to reproduce:
1. Open Audacity (with no tracks in track table).
2.  Press ALT+f, t; the Metadata Editor opens.
3.  Press ALT+l.
Actual result: JAWS says "to use this feature you must select at least one track first".
Expected result: The Load Template dialog opens.

**Comment by fergregoire94** [comment]
This does not happen in Spanish, because the script for loading presets in VST plugins, even when it is assigned to the same access key as the **Load** (**Cargar**) button in Spanish, ALT+C, pass the keystroke through to the application when no VST plugin is focused.
This does not work correctly in English because the SilenceLabels script says the message inmediately if the track table is not found with tracks.
Until this is fixed in the scripts, users can workaround this by assigning another keyboard shortcut to the Silence Labels operation and change keystroke of the SilenceLabels scripts consequently by using JAWS Keyboard Manager. To fix this from base, you should add another ElIf statement to the SilenceLabels script which test if a dialog box is active. If it helps, I tested successfully the following version of the script I modified; test was made by running Audacity in English:

```
	Script SilenceLabels ()
		If DialogActive() then
		TypeKey (GetScriptKeyName("SilenceLabels"))
	ElIf NoProject () Then
		SayNoProject ()
		Return
	ElIf FocusInMainWindow () && !IsStopped () Then
		SayNotStopped ()
	ElIf !IsTrackSelected () Then
		SayNoTrackSelected ()
		Return
	Else
		AnnounceKeyMessage (msgSilenceLabels)
	EndIf
	EndScript
```

Hope it helps!

**Comment by campg2j003** [comment]
DialogActive would work, but FocusInMainWindow should also work, and if DialogActive is needed it should be there.  The following scripts need the same change: CopyLabels SplitLabels JoinLabels  DisjoinLabels SelSyncLockTracks ZoomSel GoSelStart GoSelEnd MixAndRenderToNewTrack RepeatLastEffect ReplaceWithSilence Duplicate Trim SplitCut SplitDelete Split SplitNew Join Disjoin CutLabels DeleteLabels SplitCutLabels SplitDeleteLabels.

**Comment by campg2j003** [comment]
Fixed in cd2b723.
